sdk/daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/PreprocessorSpec.scala

```diff
@@ -147,6 +154,85 @@ class PreprocessorSpec(majorLanguageVersion: LanguageMajorVersion)
       )
     }
 
+    def testBuildPackageResolution(
+        packageMap: Seq[(String, String, String)],
+        packagePreferenceSet: Seq[String],
+    )(expected: Result[Map[String, String]]): Assertion = {
```

Comment:
Minor: why is expected part of a different parameter list?

Reply:
ah missed it. I was planning a lambda in there, but simplified and didn't uncurry it. Thanks 👍